static/src/javascripts/projects/commercial/modules/dfp/prepare-permutive.js

```diff
@@ -0,0 +1,137 @@
+// @flow
+import { loadScript } from 'lib/load-script';
+import config from 'lib/config';
+import reportError from 'lib/report-error';
+
+declare var permutive: any;
+
+/* eslint-disable */
+export const initPermutive = (): Promise<void> => {
+    return new Promise(resolve => {
+        (function (n, e, o, r, i) {
+            if (!e) {
+                (e = e || {}),
+                    (window.permutive = e),
+                    (e.q = []),
+                    (e.config = i || {}),
+                    (e.config.projectId = o),
+                    (e.config.apiKey = r),
+                    (e.config.environment =
+                        e.config.environment || 'production');
+                for (
+                    let t = [
+                            'addon',
+                            'identify',
+                            'track',
+                            'trigger',
+                            'query',
+                            'segment',
+                            'segments',
+                            'ready',
+                            'on',
+                            'once',
+                            'user',
+                            'consent',
+                        ],
+                        c = 0;
+                    c < t.length;
+                    c++
+                ) {
+                    const f = t[c];
+                    e[f] = (function(n) {
+                        return function() {
+                            const o = Array.prototype.slice.call(arguments, 0);
+                            e.q.push({ functionName: n, arguments: o });
+                        };
+                    })(f);
+                }
+            }
+        })(
+            document,
+            window.permutive,
+            'd6691a17-6fdb-4d26-85d6-b3dd27f55f08',
```

Comment:
are those API keys? if so we can also extract them in aws params

Reply:
Similar response to the previous comment 👆

Reply:
👍

Reply:
Similar to my previous comment, I agree that this should be in the parameter store.